images/other/mindustry.json

```diff
@@ -0,0 +1,82 @@
+{
+  "meta": {
+    "api_version": 2
+  },
+  "uid": "mindustry",
+  "name": "Mindustry",
+  "author": "support@endelon-hosting.de",
```

Comment:
The `author` field is mostly meant to distinguish images maintained and created by me/this project, and third-party ones. If this is to be merged and used among the other default images, then I would like the author filed to have the same value as the other images :slightly_smiling_face: